src/iOS/Xamarin.iOS/Samples/MapView/ChooseCameraController/ChooseCameraController.cs

```diff
@@ -0,0 +1,217 @@
+            sceneGraphicsOverlay.Graphics.Add(planeGraphic);
+
+            // Instantiate a new camera controller which orbits a geo element.
+            _orbitPlaneCameraController = new OrbitGeoElementCameraController(planeGraphic, 300.0)
+            {
+                CameraPitchOffset = 30,
+                CameraHeadingOffset = 150
+            };
+
+            // Instantiate a new camera controller which orbits a location.
+            _orbitCraterCameraController = new OrbitLocationCameraController(craterLocation, 6000.0)
+            {
+                CameraPitchOffset = 3,
+                CameraHeadingOffset = 150
+            };
+
+            // Set the starting camera controller.
+            _mySceneView.CameraController = _orbitPlaneCameraController;
+
+            // Add the scene to the view.
+            _mySceneView.Scene = myScene;
+        }
+
+        private void ChangeCameraController(string setting)
+        {
+            // Switch on the selected text.
+            switch (setting)
+            {
+                case "Orbit plane":
+                    // Switch to the plane camera controller.
+                    _mySceneView.CameraController = _orbitPlaneCameraController;
+                    break;
+
+                case "Orbit crater":
+                    // Switch to the crater camera controller.
+                    _mySceneView.CameraController = _orbitCraterCameraController;
+                    break;
+
+                case "Free pan":
+                    // Switch to a globe camera controller, which is free pan.
+                    _mySceneView.CameraController = new GlobeCameraController();
+                    break;
+            }
+        }
+
+        public override void LoadView()
+        {
+            // Create the views.
+            View = new UIView();
+
+            // Create the scene.
+            _mySceneView = new SceneView();
+            _mySceneView.TranslatesAutoresizingMaskIntoConstraints = false;
+
+            // Add a segmented control for selecting the camera controller.
+            _operationPicker = new UISegmentedControl("Orbit plane", "Orbit crater", "Free pan");
+            _operationPicker.TranslatesAutoresizingMaskIntoConstraints = false;
+            _operationPicker.SelectedSegment = 0;
+
+            // Toolbar for the segmented control.
+            UIToolbar toolbar = new UIToolbar();
+            toolbar.TranslatesAutoresizingMaskIntoConstraints = false;
+            toolbar.Items = new[]
+            {
+                new UIBarButtonItem {CustomView = _operationPicker}
+            };
+
+            // Add the views.
+            View.AddSubviews(_mySceneView, toolbar);
+
+            // Lay out the views.
+            NSLayoutConstraint.ActivateConstraints(new[]{
+                _mySceneView.TopAnchor.ConstraintEqualTo(View.SafeAreaLayoutGuide.TopAnchor),
+                _mySceneView.BottomAnchor.ConstraintEqualTo(View.BottomAnchor),
```

Comment:
You'll want the bottom of the sceneView constrained to the top of the toolbar to ensure that the attribution bar and license text is visible.